src/prebid.js

```diff
@@ -196,6 +197,18 @@ $$PREBID_GLOBAL$$.setTargetingForAst = function() {
   events.emit(SET_TARGETING);
 };
 
+function emitAdRenderFail(reason, message, bid) {
+  const data = {};
+
+  data.reason = reason;
+  data.message = message;
+  if (data.bid) {
```

Comment:
I wanted to check if this logic is properly configured.  It seems like this `if` condition would never be met because the `data` is initialized as empty each time the function is called, so the `data.bid` property would always be `undefined`.  Can you please take a look and clarify?

Reply:
agreed. Changing that